doc/src/build/sui-transfer-policy.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Sui Transfer Policy
+---
+
+A Sui Transfer Policy ([`TransferPolicy` object](https://github.com/MystenLabs/sui/blob/main/crates/sui-framework/docs/transfer_policy.md) specifies the conditions that must be met to transfer an item from a Sui kiosk. You must include a transfer policy to enable transfers, and the policy must be available to other users for them to purchase an item from your kiosk. When a transaction occurs on an item in a kiosk, the buyer receives a transfer request (`TransferRequest` struct). The transfer request must be confirmed in the associated transfer policy for the transaction to succeed.
+
+By default, a transfer policy is empty, and the request confirmation does not require any actions from the buyer other than calling the `confirm_request` function. However, as the owner of the transfer policy (`TransferPolicy` object), you can add any number of programmable rules that  must be completed for the request to be confirmed.
+
+**Transfer policies are:**
+ * Created using the `Publisher` object authorization
+ * Issued per type, and use only this type
+ * Enforced on all trades that occur in a kiosk
+ * Required for a type (T) to be made available for sale in a kiosks
```

Comment:
Would "...type `<T>` to be" be better? The ticks are required so may not be worth the maintenance cost.

Reply:
Yes, good call